modules/manager/config/types.go

```diff
@@ -0,0 +1,37 @@
+// This package serves to break the cyclic dependency between the manager and
+// packages that import the config types.
+
+package config
+
+import (
+	"fmt"
+)
+
+// KeyType is the type of private key.
+type KeyType string
+
+const (
+	// ECDSA is the key type for ECDSA keys.
+	ECDSA KeyType = "ecdsa"
+	// RSA is the key type for RSA keys.
+	RSA KeyType = "rsa"
+)
+
+// Set implements flag.Value.
+func (kt *KeyType) Set(v string) error {
+	switch v {
+	case string(ECDSA), string(RSA):
+		*kt = KeyType(v)
+	case "":
+		// Default to ECDSA.
+		*kt = ECDSA
```

Comment:
Please consider extracting `ECDSA` to a const similar to `DefaultClusterCAKeySize` and use it here as well as in `cli.New`.

Reply:
cea025bf6338ff96a4f0949900a8da5102c5ca0f